novawallet/Modules/Vote/Governance/Referendums/Presenter/ReferendumsPresenter.swift

```diff
@@ -35,13 +35,19 @@ final class ReferendumsPresenter {
     var countdownTimer: CountdownTimer?
     var timeModels: [ReferendumIdLocal: StatusTimeViewModel?]? {
         didSet {
-            observableState.state.timeModels = timeModels
+            observableViewState.state.timeModels = timeModels
         }
     }
 
     private(set) var filter = ReferendumsFilter.all
 
-    let observableState = Observable<ReferendumsState>(state: .init(cells: [], timeModels: nil))
+    let tinderGovObservableState = Observable<NotEqualWrapper<[ReferendumIdLocal: ReferendumLocal]>>(
```

Comment:
I would make it generic and move tinder gov specific logic (filtering/sorting) to TinderGovBuilder.

Reply:
Will present in the coming PR